### DISCUSSION TITLE
Support for RX 6600M

### DISCUSSION THREAD

**Discussion by Ahmed-Mohamed-01** [ROCm/ROCm#2652] [Q&A]
Hello ROCm Community,
I just started learning PyTorch and I found out that I can use my AMD GPU card in developing model, I installed the PyTorch Package along with ROCm but it gives me segmentation error when I try to preform any actions on the GPU, So I am wondering does ROCm support my GPU card in the first place or not? My GPU card is AMD RX 6600M.
Thank you so much

**Comment by Unknowpers** [comment]
based on my experience, there are a few cards that are supported officially but the rest of devices, can be unofficially.
i needed to get hip sdk 5.5.0 running on windows/visual studio. installed it, opened the example project and after building, got 0 on all my computing buffers on gpu. it was like nothing was being processed on gpu.
after checking settings for decades, found out that i had to insert my gpu's arch code name somewhere in a field to tell the sdk to build the app for it.

here in this link and Option3, theres a list of gfx.xxx things. thats what im referring to. you might need to add your own (gfx1032) somewhere.
https://rocm.docs.amd.com/en/latest/how_to/pytorch_install/pytorch_install.html

cant say more since have not used pytorch and am a cuda dev more than hip but anyway, this might be worth checking.
